install_scripts/app.py

```diff
@@ -369,6 +370,7 @@ def get_replicated_compose_v3_template_args(replicated_channel=None,
         # query params
         airgap=airgap,
         ca=ca,
+        cert=cert,
```

Comment:
this seems silly but i see youre just following ca here

Reply:
yep